cspell.yaml

```diff
@@ -33,6 +33,7 @@ ignorePaths:
     - webui/build
     - webui/node_modules
     - webui/lib/api/generated/**
+    - pkg/lib/license/license_validator_test.go
```

Comment:
is this needed

Reply:
@wdbaruni unfortunately cspell will complain about JWT segments which are basic64 encoded. 
Is there a better way to ignore those in cspell ?

Reply:
just fyi, you can ignore file or line.

https://cspell.org/configuration/document-settings/

Reply:
Fixed.